pkg/pillar/cmd/zedkube/lease.go

```diff
@@ -124,15 +163,31 @@ func SignalStopLeaderElection(ctx *zedkubeContext) {
 func handleControllerStatusChange(ctx *zedkubeContext, status *types.ZedAgentStatus) {
 	configStatus := status.ConfigGetStatus
 
-	log.Functionf("handleControllerStatusChange: Leader enter, status %v", configStatus)
+	//log.Functionf("handleControllerStatusChange: Leader enter, status %v", configStatus)
```

Comment:
I think LGTM, maybe this extra line could be removed though?  or leave as function and wait for end-user to change the log level config-property?

Reply:
removed the commented line. this is for PoC branch debug, when porting to mainline we can change those.